AudibleBoulders/AudibleBoulders
# Added media queries for mobile optimization for iPhone 6/6+

#### What's this PR do?
1) Added media queries for mobile optimization for iPhone 6/6+
2) Some enhancements to Desktop styling (enlarged GitSpy Icon in nav)
3) Fixed gitignore
4) Removed gitSpy folder
5) Added google fonts API for access to 'Montserrat' font. This font is used in everything that is not a 'Title'. 
6) Made enhancements for mobile. Table rows now cover 100% of the page vs 90% previously. This was done to maximize horizontal space for the repo names and delete button.
7) Setup page is now styled on mobile and desktop. Also added a back button in setup page.

#### What are the important parts of the code?
1) Application is now optimized for mobile for iPhone 6/6+

#### How should this be tested by the reviewer?
1) Pull down
2) Test in developer console for iPhone 6/6+. See that there are styling enhancements for mobile.

#### Is any other information necessary to understand this?
N/A

#### What are the relevant tickets? (Please add `closes`, `refs`, etc)
N/A

#### Screenshots (if appropriate)
N/A
